rpc/server.go

```diff
@@ -365,6 +365,14 @@ func (s *Server) readRequests(codec rpc.ServerCodec, authHook func(proto.Message
 			return
 		}
 
+		if err := authHook(args, meth.public); err != nil {
+			responses <- serverResponse{
+				req: req,
+				err: err,
+			}
+			continue
```

Comment:
Should this be continue? Is it ok to keep the connection open when the an unauthorized request comes in?
Either way, we'll need a comment to say which we pick. I'm ok with either.